structuralcodes/geometry/_circular.py

```diff
@@ -0,0 +1,173 @@
+    """
+
+    _radius: float
+
+    def __init__(
+        self,
+        size: float,
+        material: t.Union[Material, ConstitutiveLaw],
+        is_radius: bool = False,
+        n_points: int = 20,
+        density: t.Optional[float] = None,
+        concrete: bool = False,
+    ) -> None:
+        """Initialize a CircularGeometry.
+
+        Arguments:
+            size (float): The size of the geometry. By default this number
+                is assumed as the diameter of the section. To interpret this
+                as the radius `is_radius` should be set as `True`.
+            material (Union(Material, ConstitutiveLaw)): A Material or
+                ConsitutiveLaw class applied to the geometry.
+            is_radius (bool): Indicates if size is interpreted as radius
+                (default = False).
+            n_points (int): The number of points used to discretize the
+                circle as a shapely `Polygon` (default = 20).
+            density (Optional(float)): When a ConstitutiveLaw is passed as
+                material, the density can be provided by this argument. When
+                material is a Material object the density is taken from the
+                material.
+            concrete (bool): Flag to indicate if the geometry is concrete.
+
+        Note:
+            The CircularGeometry is simply a wrapper for a SurfaceGeometry
+            object.
+        """
+        # Check that size is strictly positive
+        if size <= 0:
+            raise ValueError('Size must be a positive number.')
+        # Manage size as radius or diameter (default)
+        self._radius = size if is_radius else size / 2.0
+        # Create the shapely polygon
+        polygon = _create_circle(radius=self._radius, npoints=n_points)
+        # Pass everything to the base class
+        super().__init__(
+            poly=polygon, material=material, density=density, concrete=concrete
+        )
+
+    @property
+    def radius(self):
+        """Returns the radius of the circle."""
+        return self._radius
+
+    @property
+    def diameter(self):
+        """Return diameter of the circle."""
+        return 2 * self._radius
+
+
+class CircularRCGeometry(CompoundGeometry):
+    """This is a wrapper class for defining a CompoundGeometry made of a
+    surface with circular shape and with a set of bars around the circle.
+    """
+
+    _radius: float
+
+    def __init__(
+        self,
+        size: float,
+        surface_material: t.Union[Material, ConstitutiveLaw],
+        diameter: float,
+        number: int,
+        reinforcement_material: t.Union[Material, ConstitutiveLaw],
+        cover: float,
+        is_radius: bool = False,
+        n_points: int = 20,
+        surface_density: t.Optional[float] = None,
+        concrete: bool = False,
+    ) -> None:
+        """Initialize a CircularRCGeometry.
+
+        Arguments:
+            size (float): The size of the geometry. By default this number
```

Comment:
Same comment as for `CircularGeometry`, could we do either `radius` or `diameter`?

Reply:
As agrred, I am removing CircularRC.